_posts/2018-06-13-cockpit-170.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 170
+author: pitti
+date: 2018-06-13
+tags: cockpit linux
+slug: cockpit-170
+category: release
+summary: Cockpit with many bug fixes
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.
+Here are the release notes from version 170.
+
+### Software Updates: Layout rework
+
+The header bar and updates list on the Software Updates page now work properly
+on mobile browsers. In desktop mode, the updates list now avoids line breaks
+in the version number column.
+
+![Updates Mobile](/images/updates-mobile.png)
+
+### oVirt: Use authenticated libvirt connection by default
+
+If [oVirt](https://www.ovirt.org/) has been installed without a
+machines-ovirt.config file or without a `VIRSH_CONNECTION_URI` option, the
+oVirt Machines page now uses a TLS-enabled libvirt connection URI instead of
+the previous `qemu:///`. This enables libvirt operations that require
+authentication, such as sending an NMI.
```

Comment:
I'd suggest rewriting the if/then to be more straightforward, like:

> The [oVirt](https://www.ovirt.org/) version of the Machines page now uses encryption by default. Instead of communicating with oVirt using `qemu:///`, it uses a TLS-enabled libvirt connection.
>
> A `machines-ovirt.config` file or a `VIRSH_CONNECTION_URI` option may override the TLS-based connection.
>
> Using better security enables libvirt operations which require authentication. One common example is sending an NMI.

Reply:
(...Perhaps these last two sentences should be swapped? Hmm...)